### PR TITLE
[stable/rabbitmq-ha]  add configuration for initContainer

### PR DESCRIPTION
**Pull request by PixelJonas**
#### Is this a new chart
No

#### What this PR does / why we need it:
I'm currently using NFS mounts for my cluster. running as `root` and changing permissions does not work using these volumes. This resulted in the rabbitmq initContainer always crashing.

This PR adds multiple options to configure the initContainer including the default `securityContext` options and an extra option to disable the `chown` when using `runAsGroup` and/or `fsGroup`.

#### Which issue this PR fixes
none

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/rabbitmq-ha/templates/statefulset.yaml [hunk 1]
```diff
@@ -66,13 +66,14 @@ spec:
...
       initContainers:
+{{- if .Values.initContainer.enabled }}
         - name: bootstrap
           image: {{ .Values.busyboxImage.repository}}:{{ .Values.busyboxImage.tag}}
           imagePullPolicy: {{ .Values.busyboxImage.pullPolicy }}
           securityContext:
-            runAsGroup: 0
-            runAsNonRoot: false
-            runAsUser: 0
+            runAsGroup: {{ .Values.initContainer.runAsGroup }}
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/templates/statefulset.yaml]: Would prefer setting entire security context as yaml:
```yaml
securityContext:
  {{- toYaml .Values.initContainer.securityContext | nindent X }}
```

File: stable/rabbitmq-ha/templates/statefulset.yaml [hunk 2]
```diff
@@ -85,7 +86,7 @@ spec:
               touch "${RABBITMQ_MNESIA_DIR}/force_load"
             fi
             {{- end }}
-            {{- if and (.Values.securityContext.runAsUser) (.Values.securityContext.fsGroup) }}
+            {{- if and (.Values.securityContext.runAsUser) (.Values.securityContext.fsGroup) (not .Values.initContainer.skipChown) }}
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/templates/statefulset.yaml]: Would prefer negated property `chownFiles=true`

File: stable/rabbitmq-ha/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.44.2
+version: 1.44.3
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: Minor version bump since we have some new properties: 1.45.0

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
/assign
/ok-to-test

**Comment by PixelJonas** [comment]
@steven-sheehy sorry it took me long but I resolved your comments.

**Review by steven-sheehy — APPROVED** [review]
/lgtm

**Pull request merged**